rollup.config.js

```diff
@@ -1,11 +1,26 @@
-import buble from 'rollup-plugin-buble'
+const pkg = require('./package.json')
```

Comment:
please use the modules syntax here instead of commonjs

Reply:
Ok I didn't know that also because I have never imported the `package.json` in a build but it's fair enough thanks for the explanation